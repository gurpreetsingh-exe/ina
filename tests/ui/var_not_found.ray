// COMPILE_FAIL

fn main() {
    a;
}
