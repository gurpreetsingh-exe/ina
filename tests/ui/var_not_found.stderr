error: local variable `a` is not found in this scope
   --> var_not_found.ray:4:5
    |
 LL |     a;
    |     ^ not found in this scope

